### PR TITLE
added cypress directories to node ignore

### PR DESCRIPTION
**Pull request by chrisenitan**
**Reasons for making this change:**

1. I've seen a lot like mine and my teams projects testing with [Cypress io](https://www.cypress.io/) and thought best to have that included in the Node ignore list. 
    - [>17k Repo](https://github.com/search?q=cypress&type=repositories)
2. I also look around other similar test frameworks like Test cafe and Cypress seems to be the only one creating custom files at runtime. 

**Links to documentation supporting these rule changes:**

- According to the Cypress [official documentation](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests), cypress generates sample tests in the integration folder
- `cypress.env.json` is an alternative to cypress.json which acts as similar to dotenv [Documentation](https://docs.cypress.io/guides/guides/environment-variables#Option-1-configuration-file)
- Other asset files "report, videos, screenshots" are mostly ignored but optional. 

If this is a new template:

- Not a new template but there are standalone Cypress projects that may require a Cypress template. Please let me know what you think.

### CODE REVIEW

File: Node.gitignore [hunk 1]
```diff
@@ -26,6 +26,14 @@ coverage
 # nyc test coverage
 .nyc_output
 
+#cypress
+cypress/integration/1-getting-started/
+cypress/integration/2-advanced-examples/
```
Comment (bdougie) [inline Node.gitignore:31]: These files seem specific to a tutorial or guide. They are not listed in the documentation you provided.

**Review by bdougie — CHANGES_REQUESTED** [review]
Remove the tutorial files.

File: Node.gitignore (hunk 1 above)
Comment (chrisenitan) [reply Node.gitignore:31]: - Could not find any documentation on Cypress seeding the project with these files but they are scaffolded once you run Cypress initially. 
- You do have options to remove them, so adding the dir to the ignore list might not always be useful. 

What do you think? 
Screenshot 2021-12-05 at 12 50 16

Comment (chrisenitan) [reply Node.gitignore:31]: - Could not find any documentation on Cypress seeding the project with these files but they are scaffolded once you run Cypress initially. 
- You do have options to remove them, so adding the dir to the ignore list might not always be useful. 
- However I don't assume anyone cares to stage those. 

What do you think? 
Screenshot 2021-12-05 at 12 50 16

Comment (bdougie) [reply Node.gitignore:31]: I still question the number of files added here. Could this instead be `cypress/` and `!cypress/whatever` the files needed?

cc @bahmutov in there is some context missing.

**Comment by wirecat** [comment]
I believe this PR has gone stale and I'm going to close for now @kannanputhuval.
Please feel free to re-open if something changes.

**Pull request closed without merging**